extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -1002,7 +1000,11 @@ public boolean isGroupCoordinator() {
 
     @Override
     public String getUDN() {
```

Comment:
This was to avoid checking the thing  config each time. But you are right, i can suppress the udn member and check the thing config each time. Will fix that.